scripts/publish-image.sh

```diff
@@ -27,14 +27,18 @@ if [ -z "${CTR_TAG}" ]; then
 fi
 
 if [[ "$VERIFY_TAGS" == "true" ]]; then
-    tokenUri="https://auth.docker.io/token?service=registry.docker.io&scope=repository:$IMAGE_REPO/$IMAGE_NAME:pull"
+    image="$IMAGE_NAME"
+    if [[ $OS == "windows" ]]; then
+      image="$image-windows"
+    fi
```

Comment:
Why can't the caller (Makefile) pass the correct IMAGE_NAME?

Reply:
We still use the image name when we do `make ARGS=--push "docker-build-windows-$IMAGE_NAME"`.

Basically, this is the smallest change I could think of to get this to work correctly. There are plenty of things we can do to clean all of the image building later, but I just wanted to unblock the release for now.